src/lib/platform.js

```diff
@@ -0,0 +1,72 @@
+var $builtinmodule = function(name){
+    var mod = {};
+    var inBrowser = (typeof window != "undefined") && (typeof window.navigator != "undefined");
+    mod.python_implementation = new Sk.builtin.func(function() {
+        // This checks that the correct number of arguments were passed
+        Sk.builtin.pyCheckArgs("python_implementation", arguments, 0, 0);
+        return new Sk.builtin.str("Skulpt");
+    })
+    mod.node = new Sk.builtin.func(function() {
+        Sk.builtin.pyCheckArgs("node", arguments, 0, 0);
+        return new Sk.builtin.str("");
+    })
+    mod.version = new Sk.builtin.func(function() {
+        var vers;
+        Sk.builtin.pyCheckArgs("version", arguments, 0, 0);
+        if (Sk.python3) {
+            vers = "3.2.0";
```

Comment:
Because 3.3 introduced Syntax changes that we don't have (like yield).  Perhaps 3.0.0 is a better choice, though...  Not sure.